app/models/form_attachment.rb

```diff
@@ -36,15 +36,15 @@ def get_file
   private
 
   def unlock_pdf(file, file_password)
-    return file unless File.extname(file) == '.pdf'
+    return file unless File.extname(file).downcase == '.pdf'
```

Comment:
What if this line is removed all together?
With a method name of `unlock_pdf`, it should already be determined that it's a PDF before it's called. 
Ideally, it should be the responsibility of the caller to make sure it's a PDF before calling this method. (That way they can do a better job than checking the file extension...)

Reply:
You could move the type check up to line 16 where I think it makes more sense, logically.

Reply:
Relocated extension type check per suggestion. Thanks, @shaunburdick.